core/src/components.d.ts

```diff
@@ -2535,6 +2535,10 @@ export namespace Components {
           * An array of buttons for the toast.
          */
         "buttons"?: (ToastButton | string)[];
+        /**
+          * If `true`, the text of the toast will be centered.
+         */
+        "centeredText": boolean;
```

Comment:
This prop should not be needed. If we just set `text-align: center` on `:host` in `toast.ios.scss` that should be enough. Users can override it if they want the text aligned another way.

Reply:
Actually not sure how this affects buttons, but either way we shouldn't need a prop to do this.

Reply:
@liamdebeasi Buttons are centered too.

Reply:
@liamdebeasi See commit dc709a5.
The property **centeredText** has been removed. Text will be centered automatically and the font will be bold on iOS.

Reply:
> @liamdebeasi Buttons are centered too.

Do you know if they are centered on native iOS 14 apps?